src/components/templates/PartyMap/components/Map/PartyMapRoomOverlay.tsx

```diff
@@ -14,104 +17,205 @@ import RoomAttendance from "../RoomAttendance";
 interface PartyMapRoomOverlayProps {
   // Passed down from Map component
   room: PartyMapRoomData;
+  enterRoom: (room: PartyMapRoomData) => void;
 
   // Passed down from Camp component (via Map component)
   venue: PartyMapVenue;
   attendances: Attendances;
   selectedRoom?: PartyMapRoomData;
-  setSelectedRoom: (room?: PartyMapRoomData) => void;
-  setIsRoomModalOpen: (value: boolean) => void;
+  selectRoom: (room: CampRoomData) => void;
```

Comment:
This should be PartyMapRoomData

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/840/commits/a82b60e311816413cd5a4ad72036ceab1e20740b